alchemist-loading/src/test/kotlin/it/unibo/alchemist/test/TestCSVExporter.kt

```diff
@@ -24,24 +26,53 @@ import java.io.File
 
 class TestCSVExporter<T, P : Position<P>> : StringSpec({
     "test exporting data on CSV file" {
-        val file = ResourceLoader.getResource("testCSVExporter.yml")
-        assertNotNull(file)
-        val loader = LoadAlchemist.from(file)
-        assertNotNull(loader)
-        val initialized: InitializedEnvironment<T, P> = loader.getDefault()
-        val simulation = Engine(initialized.environment)
-        simulation.addOutputMonitor(GlobalExporter(initialized.exporters))
-        simulation.play()
-        simulation.run()
+        val initialized: InitializedEnvironment<T, P> = loadAlchemist("testCSVExporter.yml")
+        runSimulationUsingExporters(initialized)
         initialized.exporters.size shouldBe 1
-        val exporter = initialized.exporters.first()
-        require(exporter is CSVExporter) {
-            "Invalid exporter type '${exporter::class.simpleName}'"
-        }
+        val exporter = getCSVExporter(initialized)
         val outputFile = File(exporter.exportPath)
             .listFiles()
             ?.find { it.name.startsWith("00-testing_csv_export_") && it.extension == exporter.fileExtension }
         outputFile.shouldNotBeNull()
         outputFile.shouldExist()
     }
-})
+    "test on column alignment" {
+        val initialized: InitializedEnvironment<T, P> = loadAlchemist("testCSVExportColumnAlignment.yml")
+        runSimulationUsingExporters(initialized)
+        val exporter = getCSVExporter(initialized)
+        // Get the first line of the output produce by CSVExporter
+        val exporterFirstLine = File(exporter.exportPath)
+            .listFiles()?.first { it.name.startsWith("column-alignment") }
+            ?.readLines()
+            ?.dropWhile { !it.contains("d c b a") } // remove the lines before the column names
+            ?.drop(1) // I am not interested in column head
+            ?.first()
+        exporterFirstLine.shouldNotBeNull()
+        exporterFirstLine.shouldNotBeEmpty()
+        exporterFirstLine.shouldContain("0 1 2 3")
+    }
+}) {
+    /* common utility functions */
+    companion object {
```

Comment:
These could actually be added to alchemist-test, or maybe they are already there in some form, and only need to get enhanced